webpack/loaders.js

```diff
@@ -5,7 +5,7 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 exports.angular = { // ships in ES6 format now
   test: /\.js$/,
-  loader: 'babel-loader',
+  use: ['babel-loader'],
```

Comment:
It does complain for the babel one in particular, had to move to a verbose loader object